script.js

```diff
@@ -0,0 +1,54 @@
+let sections = document.getElementsByTagName('section')
+let actionButtonNext = document.querySelector('.action-buttons__next')
+let actionButtonBack = document.querySelector('.action-buttons__back')
+let actionButtonsDiv = document.querySelector('.action-buttons')
+
+//Hide all sections except first one
+for (let i = 1; i < sections.length; i++) {
+  sections[i].classList.add('visually-hidden')
+}
+
+//Set the event to the Next Button to change between sections
+let globalActiveSectionNumber = 0
+updateButtonStatus()
+actionButtonNext.addEventListener('click', () => {
+  globalActiveSectionNumber++
+  changeSection(true)
+  updateButtonStatus()
+})
+
+//Set the event to the Previous Button to change between sections
+actionButtonBack.addEventListener('click', () => {
+  globalActiveSectionNumber--
+  changeSection(false)
+  updateButtonStatus()
+})
```

Comment:
what if instead of this you have something like

```
val onNavigationButtonClicked = (sectionNumberChange) => {
  globalActiveSectionNumber += sectionNumberChange
  changeSection(sectionNumberChange > 0)
  updateButtonStatus()
})
actionButtonNext.addEventListener('click', () => {
  onNavigationButtonClicked(1)
})

//Set the event to the Previous Button to change between sections
actionButtonBack.addEventListener('click', () => {
  onNavigationButtonClicked(-1)
})
```